client-templates/nexus3/accept.json.sample

```diff
@@ -0,0 +1,26 @@
+{
+  "public": [
+    {
+      "method": "any",
+      "path": "/*",
+      "stream": true
+    }
+  ],
+  "private": [
+    {
+      "method": "HEAD",
+      "path": "/",
+      "origin": "${BASE_NEXUS_URL}"
+    },
+    {
+      "method": "GET",
+      "path": "/service/rest/v1/status/check",
+      "origin": "${BASE_NEXUS_URL}"
+    },
+    {
```

Comment:
should we add on the `README.md` explanation for running in unauthenticated mode (without `/check`), using: 
```
 {
      "method": "GET",
      "path": "/service/rest/v1/status",
      "origin": "${BASE_NEXUS_URL}"
 }

Reply:
we can't add notes to accept.json, do you mean on the `README.md `as well as on the `.env` file? 
and the empty path is for commentary purposes, to emphasize the `/check` diff.

Reply:
Yeah I meant a note somewhere, not on the json. Maybe in the README or we could make it clear in the `.env` file comments instead? Im not sure I understand your comment about the empty path. I was referring to use not being able to do this:
```
 {
      "method": "GET",
      "path": "",
      "origin": "BROKER_CLIENT_VALIDATION_URL"
 }
```

Reply:
Also I meant we shouldnt leave both `/service/rest/v1/status/check` and `/service/rest/v1/status` in here. Its technically wrong.